### PR TITLE
Flex 2.6.4-1 => 2.6.4-2

### PR DESCRIPTION
**Pull request by uberhacker**
Add missing `lex` command.

### CODE REVIEW

File: packages/flex.rb [hunk 1]
```diff
@@ -3,29 +3,35 @@
...
+    downloader 'https://gitlab.archlinux.org/archlinux/packaging/packages/flex/-/raw/main/flex-pie.patch?ref_type=heads&inline=false',
+               '20f3cce6b0ea6ab67a902a46b89c292b959994dedcbe6ee5d187f9bba1408b0e', 'flex-pie.patch'
     system 'patch -p1 -i flex-pie.patch'
     system 'autoreconf -fiv'
   end
 
-  configure_options '--with-pic --disable-static --enable-shared'
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/flex", "#{CREW_DEST_PREFIX}/bin/lex"
```
[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/flex.rb:35]: Yes, the `lex` command is needed to successfully compile nethack. :smiley_cat:

[the remark this answers is not in the document]
Comment (Zopolis4) [reply packages/flex.rb:35]: Wouldn't that be a nethack issue?

[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/flex.rb:35]: What does it hurt to add the command?  No other package provides it.

[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/flex.rb:35]: FWIW, the Debian flex package also includes the `lex` executable.

[the remark this answers is not in the document]
Comment (uberhacker) [reply packages/flex.rb:35]: As a matter of fact, see below:
```
$ ls -l /usr/bin/lex
lrwxrwxrwx 1 root root 4 May 18  2023 /usr/bin/lex -> flex
```

[the remark this answers is not in the document]
Comment (Zopolis4) [reply packages/flex.rb:35]: Fair enough.

**Review by Zopolis4 — APPROVED** [review]

**Pull request merged**